Examples for JCGS/BF.PT.uni.R

```diff
@@ -0,0 +1,214 @@
+
```

Comment:
All files in JCGS should go to a different repository.

Reply:
Ok.